### PR TITLE
Feature/spawner improvements

### PR DESCRIPTION
**Pull request by s7ubborn**

### CODE REVIEW

File: Assets/Scripts/ODYSSEY/Features/Minimap/Scripts/Minimap.cs [hunk 1]
```diff
@@ -190,6 +190,9 @@ public void WorldObjectSpawned(WorldObject worldObj)
     {
         if (!worldObj.showOnMiniMap) return;
 
+        string indicatorKey = worldObj.guid.ToString();
+        if (!_objectIndicators.ContainsKey(indicatorKey)) return;
```
Comment (jor-rit) [inline Assets/Scripts/ODYSSEY/Features/Minimap/Scripts/Minimap.cs:194]: isn't this the wrong way around?
if it already exists, return. If it doesn't exist, continue to add it.

**Review by jor-rit — COMMENTED** [review]
And now that I think about this, this happens when a object gets updated from the backend. So maybe we should remove & add the new one?
(did not look into what actually can be changed in this minimap code, but that feel safer. Not sure what happens when the 'name' of an object changes, for example)

File: Assets/Scripts/ODYSSEY/Features/Minimap/Scripts/Minimap.cs (hunk 1 above)
Comment (s7ubborn) [reply Assets/Scripts/ODYSSEY/Features/Minimap/Scripts/Minimap.cs:194]: Yes, you are correct

**Comment by s7ubborn** [comment]
I am going to go ahead and merge this, with your comments addressed @jor-rit

**Pull request merged**